vars/goMake.groovy

```diff
@@ -10,6 +10,8 @@ def call(body) {
     def ghOrg =  config.githubOrganisation
     def dockerOrg = config.dockerOrganisation
     def prj = config.project
+    def makeCommand = config.makeCommand ?: 'make'
+    def installPreRequisites = config.installPreRequisites
 
```

Comment:
I am sure this is not the best approach but wasn't about any better approach, let me know if we can do here something better.